README.md

```diff
@@ -43,19 +43,23 @@ The Docker environment is recommended because of the fewer setup steps required
  * Install [git](https://git-scm.com/downloads)
  * Install [Git-LFS](https://git-lfs.github.com/)
 
-### Cloning the repo
+### Cloning the repo for contribution
 
-Clone the docs repo into a directory you want the ODK-X docs files to be located. For example, navigate to the directory you want the files to be located using the "cd" (Change Directory) command on the command line:.
+Clone the docs repo into a directory you want the ODK-X docs files to be located. Below are the steps required to clone the docs repo:
+1. Visit this link https://github.com/odk-x/docs on your browser to locate the docs repo
+2. Find and click the "fork" button to create a personal fork of the project on Github
+3. Navigate to the directory you want the files to be located using the "cd" (Change Directory) command on a command-line interface on your local machine:
 ```
 cd <DIRECTORY>
 ```
+4. Then use git to get a copy of the ODK-X documentation files by Cloning the fork to your local machine from your github account using:  
```

Comment:
Cloning caps unnecessary
colon does not seem to be followed by command, maybe because additional text was added?